Dockerfiles/build-ebpf-objects.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.18
+# clang and llvm are needed by bpf2go.
+# gcc-multilib is needed for <asm/types.h>.
+# libelf-dev is needed to compile libbpf.
+RUN apt-get update && apt-get install -y clang llvm gcc-multilib libelf-dev
+# Let's install cutting edge libbpf.
+RUN git clone https://github.com/libbpf/libbpf.git && cd libbpf/src && mkdir build && OBJDIR=build make install -j
+WORKDIR /work
+# The resulting docker image should be run with:
+# docker run --rm -t --name ebpf-object-builder -v/path/to/inspektor-gadget:/work name-of-this-image
+ENTRYPOINT make ebpf-objects
```

Comment:
Wouldn't this only generate bpf objects for a single target like amd64? or the arm64 support is handled differently?